tools/fluent/basic_syntax.md

```diff
@@ -0,0 +1,131 @@
+# Introduction to Fluent syntax
+
+<!-- toc -->
+
+## Basic syntax
+
+A *message* represents the basic unit of translation in Fluent. This is the simplest form a message could take:
+
+```PROPERTIES
+welcome-msg = Hello, World!
+```
+
+`welcome-msg` is the message identifier (or message ID), and `Hello, World!` is the **value** of this message.
+
+A message can contain **external arguments**, for example:
+
+```PROPERTIES
+welcome-msg = Welcome { $user }
+```
+
+`$user` is the name of the argument, and should never be translated. The fragment included between curly braces is called a **placeable**, and can be moved within the text. For example, the string above would be translated in Italian as follows, leaving the placeable unchanged:
+
+```PROPERTIES
+welcome-msg = Benvenuto { $user }
+```
+
+Messages can also reference other messages. In this case, the placeable won’t include the `$` sign, simply the message identifier:
+
+```PROPERTIES
+menu-settings = Settings
+help-menu-settings = Click “{ menu-settings }” to save the file.
```

Comment:
This might suggest that you also need `""` around the placeable. Can you either drop the quotes or provide the expected output of the `help-menu-settings` message?

Reply:
That's actually why I used curly quotes, but I see the risk. Let's drop them

Reply:
Yeah, I noticed they weren't straight double quotes and I certainly appreciate the touch :), but I think it's safer to drop them.